extra/restart-datadog-agent.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [[ $DISABLE_DATADOG_AGENT ]]; then
+  echo "DISABLE_DATADOG_AGENT environment variable is set, not starting the agent."
+  exit 0
+fi
+
+# Update integration directory configuration
+sed -i -e"s/^.*UNIX_CONFIG_PATH =.*$/UNIX_CONFIG_PATH = '\/app\/.apt\/opt\/datadog-agent\/agent'/" /app/.apt/opt/datadog-agent/agent/config.py
```

Comment:
but this directory already exists on our dynos with all the example yaml files in it and the `auto_conf` directory

```
/app/.apt/etc/dd-agent/conf.d
```

Are you sure it doesn't work if we just put it there?